src/Dashboard/Dashboard.js

```diff
@@ -1,36 +1,98 @@
 import React, { useState, useEffect } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
 import { Container, Col, Row, ProgressBar, Image } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios';
 import RegistryCard from './RegistryCard';
 import InvitationCard from './InvitationCard';
 import GuestListCard from './GuestListCard';
 import { BAlexBrushH1 } from '../Common/styled/textstyles/AlexBrushH1';
 import { BFaustinaH3 } from '../Common/styled/textstyles/FaustinaH3';
-import BlueButton from '../Common/styled/buttonstyles/BlueButton';
-import CardFormat from '../Common/styled/cardstyles/cardformat';
+import { BFaustinaH2 } from '../Common/styled/textstyles/FaustinaH2';
 
 export default function Dashboard(props) {
   const [firstName, setFirstName] = useState('John');
   const [partnerFirstName, setPartnerFirstName] = useState('Jane');
   const [daysLeft, setDaysLeft] = useState('2');
+  const [progress, setProgress] = useState(0);
+
+  const [numGuests, setNumGuests] = useState(0);
+  const [numGifts, setNumGifts] = useState('');
```

Comment:
would numGifts default state also be a number state like numGuests? just a minor comment that doesn't seem to affect any flow or anything